page.codeberg.terratactician_expandoria.game.yaml

```diff
@@ -0,0 +1,79 @@
+---
+app-id: "page.codeberg.terratactician_expandoria.game"
```

Comment:
_ in a host name is not valid, looks like codeberg allows invalid host names. You won't be able to verify this.

Reply:
https://datatracker.ietf.org/doc/html/rfc1123#page-13

> A "name" (Net, Host, Gateway, or Domain name) is a text string up
   to 24 characters drawn from the alphabet (A-Z), digits (0-9), minus
   sign (-), and period (.). 

> The syntax of a legal Internet host name was specified in [RFC-952](https://datatracker.ietf.org/doc/html/rfc952)
      [DNS:4].  One aspect of host name syntax is hereby changed: the
      restriction on the first character is relaxed to allow either a
      letter or a digit.

Reply:
I can allow an exception so that it can be built, but you won't be able to mark this as verified on the flathub website.

Reply:
We've decided to rename the organisation to `terratactician-expandoria`.